### PR TITLE
✨ Users search in PO center

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Create a minimal GUI to display users found on a given email or email part.

- ✨ Adds a new page in the PO-center to search users by email 
image
- 🎨 Adds `invitedBy` and `products` to `UserProfile` returned by search

## Related issue/s

- Implements first part of #5507 

## How to test

```cmd
cd services/web/server
make install-dev
pytest -vv tests/units/with_db -k test_search_and_pre_registration
```

- set user as PRODUCT_OWNER
- open PO-Center in UI
- select Users page

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
It can go in, I'll refactor it once it's merged.

File: services/static-webserver/client/source/class/osparc/po/Users.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2023 IT'IS Foundation, https://itis.swiss
```
Comment (odeimaiz) [inline services/static-webserver/client/source/class/osparc/po/Users.js:8]: :)

**Comment by pcrespov** [comment]
> It can go in, I'll refactor it once it's merged.

refactor? you do not like my minimalist GUI? ;-)

**Review by GitHK — APPROVED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Comment by odeimaiz** [comment]
> > It can go in, I'll refactor it once it's merged.
> 
> refactor? you do not like my minimalist GUI? ;-)

I meant creating a Base class for not copy&pasting the invitations page ;), but thanks for the cleanup!

**Pull request merged**